src/classes/rests/WheatherCallout.cls

```diff
@@ -0,0 +1,18 @@
+/**
+ * Created by Mariya on 29.11.2022.
+ */
+
+public with sharing class WheatherCallout {
+
+    public static String makeCallout() {
+        HttpRequest req = new HttpRequest();
+        String APIKey = 'c840127ab41a64ffc77361a9d11b3ab3';
```

Comment:
MAJOR
We cannot hardcode sensitive data in the code as it creates security breaches. 
Such data has to be stored as metadata, could be encrypted.
Please use Named Credentials to store API Key:
https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_callouts_named_credentials.htm